google/cloud/forseti/services/model/importer/importer.py

```diff
@@ -289,6 +293,12 @@ def run(self):
                     post_action=self._store_gsuite_membership_post
                 )
 
+                self.model_action_wrapper(
+                    self.session,
+                    inventory.iter(groups_settings_list, with_parent=True),
```

Comment:
You don't need the parent object because the settings resource includes the email address of the group.